<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QMdiArea" name="mdiArea"/>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTextEdit" name="logViewer">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextEditorInteraction</set>
        </property>
       </widget>
       <widget class="NetManager" name="netManager">
        <property name="indentation">
         <number>20</number>
        </property>
        <property name="animated">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="headerVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="headerShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Net</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/media/icons/view-grid.png</normaloff>:/media/icons/view-grid.png</iconset>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Pin</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/media/icons/merge.png</normaloff>:/media/icons/merge.png</iconset>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Session"/>
    <addaction name="actionSave_Session"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionESS_Help"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_ESS"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuAVR">
    <property name="title">
     <string>AVR</string>
    </property>
    <addaction name="actionLoad_Firmware"/>
    <addaction name="actionReload_Firmware"/>
    <addaction name="separator"/>
    <addaction name="actionStart"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionNetManager"/>
    <addaction name="actionConsole"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAVR"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_Firmware"/>
   <addaction name="actionReload_Firmware"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionNetManager"/>
   <addaction name="actionConsole"/>
  </widget>
  <action name="actionOpen_Session">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/media/icons/document-open-folder.png</normaloff>:/media/icons/document-open-folder.png</iconset>
   </property>
   <property name="text">
    <string>Open Session</string>
   </property>
  </action>
  <action name="actionSave_Session">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/media/icons/document-save.png</normaloff>:/media/icons/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save Session</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/media/icons/application-exit.png</normaloff>:/media/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionESS_Help">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/media/icons/help-contents.png</normaloff>:/media/icons/help-contents.png</iconset>
   </property>
   <property name="text">
    <string>ESS Help</string>
   </property>
  </action>
  <action name="actionAbout_ESS">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/media/icons/help-about.png</normaloff>:/media/icons/help-about.png</iconset>
   </property>
   <property name="text">
    <string>About ESS</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/media/icons/help-about.png</normaloff>:/media/icons/help-about.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="statusTip">
    <string comment="Sarp" extracomment="Stuff About Qt"/>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionLoad_Firmware">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/media/icons/document-open-folder.png</normaloff>:/media/icons/document-open-folder.png</iconset>
   </property>
   <property name="text">
    <string>Load Firmware</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/media/icons/media-playback-pause.png</normaloff>:/media/icons/media-playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/media/icons/media-playback-stop.png</normaloff>:/media/icons/media-playback-stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionReload_Firmware">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/media/icons/view-refresh.png</normaloff>:/media/icons/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Reload Firmware</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/media/icons/media-playback-start.png</normaloff>:/media/icons/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/media/icons/utilities-terminal.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Console</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Debug Console&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionNetManager">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/media/icons/code-class.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Net Manager</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Net List Manager&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NetManager</class>
   <extends>QTreeWidget</extends>
   <header>netmgr.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
